octobot_trading/exchanges/connectors/ccxt/ccxt_adapter.py

```diff
@@ -42,6 +42,22 @@ def parse_order(self, fixed, **kwargs):
         # CCXT standard order parsing logic
         return fixed
 
+    def adapt_quantities_with_contract_size(self, order_or_trade, symbol):
+        if self.connector.exchange_manager.is_future:
+            symbol = symbol or order_or_trade.get(ecoc.SYMBOL.value)
+            if symbol is None:
+                # can't get contract size
+                return
+            # amount is in contacts, multiply by contract value to get the currency amount (displayed to the user)
+            contract_size = self.connector.get_contract_size(symbol)
```

Comment:
It's a REST call or it's using market status?

Reply:
Its using market statuses

Reply:
Perfect thanks